modules/world/evaluation/evaluator_zip_merge_chn.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2020 fortiss GmbH
+//
+// This work is licensed under the terms of the MIT license.
+// For a copy, see <https://opensource.org/licenses/MIT>.
+
+#include "modules/world/evaluation/evaluator_zip_merge_chn.hpp"
+
+#include "modules/world/evaluation/labels/agent_at_lane_end_label_function.hpp"
+#include "modules/world/evaluation/labels/agent_beyond_point_label_function.hpp"
+#include "modules/world/evaluation/labels/agent_near_label_function.hpp"
+#include "modules/world/evaluation/labels/behind_of_label_function.hpp"
+#include "modules/world/evaluation/labels/ego_beyond_point_label_function.hpp"
+#include "modules/world/evaluation/labels/left_of_label_function.hpp"
+#include "modules/world/evaluation/labels/preceding_agent_label_function.hpp"
+
+namespace modules {
+namespace world {
+namespace evaluation {
+const char EvaluatorZipMergeChn::formula_[] =
```

Comment:
add descr.

Reply:
we will provide a link once the paper got accepted. add some small doc string, then it should be okay for now.